supernovacontroller/sequential/i3c.py

```diff
@@ -185,10 +186,10 @@ def init_bus(self, voltage: int=None, targets=None):
         # TODO: Toggle IBIs off
 
         status = responses[0]["result"]
-        if status == "I3C_BUS_INIT_SUCCESS":
+        if status == "DAA_SUCCESS":
```

Comment:
Shouldn't we also check that response["descriptor"]["errors"] = "NO_TRANSFER_ERROR" ?